tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./node_modules/carbon-factory/tsconfig.json",
+  "include": [
+    "src"
```

Comment:
does having this config mean you need to define the path at all in the script command?